NickvisionCavalier.GNOME/Controls/AddProfileDialog.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using static NickvisionCavalier.Shared.Helpers.Gettext;
+
+namespace NickvisionCavalier.GNOME.Controls;
+
+/// <summary>
+/// A dialog for showing a message
+/// </summary>
+public class AddProfileDialog
+{
+    private readonly Adw.MessageDialog _dialog;
+
+    public string ProfileName;
```

Comment:
`public string ProfileName { get; set; }`

Remember: 
- public variables need to be Properties (with a get and set as they behind the scenes will have a backing instance variable for encapsulation)
    - This is good because let's say you want a variable to be read only or write only you can specify only the get or set like so: `public string X { get; }` now you can only read it or `public string X { get; private set; }` will let you read it everywhere and only edit it in the same class 
- private variables don't need the get and set

Reply:
+ Add docs